microsetta_interface/templates/sample_results.jinja2

```diff
@@ -84,6 +85,8 @@
 <div class="row">
   <div class="column" style="text-align: center">
     <h3> What is in your sample? </h3>
+    <br>
+    <p>The table below shows the relative abundances of all of the organisms we observed in your sample. To produce this, we took the DNA sequences from your sample, and compared them against publicly available annotated reference databases. In some cases, a sequence may uniquely match a database record, in other cases a sequence may match may different records. To handle the uncertainty that can arise, we rely on a well used and publicly available microbiome software package called <a href="https://qiime2.org" target="_blank">QIIME 2</a>. The exact approach taken is highlighted in the main tutorial on <a href="https://docs.qiime2.org/2020.6/tutorials/moving-pictures/#taxonomic-analysis" target="_blank">feature-classification</a>.</p>
```

Comment:
Can this be split onto multiple lines? Some reasonable line width like 120 chars would be nice.